faststream/nats/message.py

```diff
@@ -17,6 +17,11 @@ async def ack(self) -> None:
             await self.raw_message.ack()
         await super().ack()
 
+    def ack_sync(self) -> None:
```

Comment:
Sorry, didn't see this first time. `ack_sync` doesn't mean, that it is a sync method. It means, that it waits for acknowledgement confirmation frame. Please, make it `async def` with original **nats-py** method awaiting.